toolbox/githubctl/main.go

```diff
@@ -303,6 +277,39 @@ func DailyReleaseQualification(baseBranch *string) error {
 	return fmt.Errorf("release qualification failed")
 }
 
+func traverseJobTree(job config.Postsubmit, postsubmitJobs *[]string, targetBranch string) {
+	for _, branch := range job.Brancher.Branches {
```

Comment:
you want to use this instead 
https://github.com/kubernetes/test-infra/blob/master/prow/config/jobs.go#L194

Reply:
Done